getting_started/topics/standalone/proc-installing-server-product.adoc

```diff
@@ -9,27 +9,26 @@ For this sample instance of {project_name}, this procedure involves installation
 
 . Go to the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?downloadType=distributions&product=core.service.rhsso[Red Hat customer portal].
 
-. Download the {project_name} Server: *rh-sso-{project_version}.zip*
+. Download the {project_name} Server: *rh-sso-7.4.0.zip*
 
 . Place the file in a directory you choose.
 
-. Unpack the ZIP file using the appropriate `unzip` utility, such as jar, tar, or unzip.
+. Unpack the ZIP file using the appropriate `unzip` utility, such as unzip, tar, or Expand-Archive.
 
 +
 .Linux/Unix
 [source,bash,subs=+attributes]
 ----
-$ unzip rhsso-{project_version}.zip
+$ unzip rhsso-7.4.0.zip
 
 or
 
-$ tar -xvzf rh-sso-{project_version}.tar.gz
+$ tar -xvzf rh-sso-7.4.0.tar.gz
```

Comment:
Why replace the SSO version variable with a hardcoded version? That seems like it'd introduce more maintenance work.

Looking at document-attributes-product, I see project_version is `7.4.0.GA`, so maybe we need to add `project_versionFiles` as `7.4.0`?